src/sass/partials/_404-page.scss

```diff
@@ -0,0 +1,71 @@
+.error-404 {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row-reverse;
+    justify-content: center;
+    align-items: center;
+    gap: 50px;
+    padding: 50px;
+  
+    @media screen and (min-width: 986px) {
```

Comment:
I quess you want to create media query for table with 768px but it was mistake with number 9 😅

Reply:
No Krystian it was not mistake if you see how it look like in the 986px breakk point you will understand my decision.